src/commands/sync/compare.ts

```diff
@@ -39,19 +39,21 @@ async function compareHandler(this: Command, pattern: string) {
 
   console.log('Your code project and Tolgee project are out of sync.');
   if (diff.added.length) {
-    console.log(ansi.green.bold(`${diff.added.length} new strings`));
+    const key = diff.added.length === 1 ? 'key' : 'keys'
+    console.log(ansi.green.bold(`${diff.added.length} new ${key} found in your code project`));
```

Comment:
the "in your code project" doesn't sound right for me, it should be "in your project code", however I think it's maybe unnecessary to write it at all "10 new keys found" is understandable, I think

Reply:
I use "code project" in contrast to the "localization project" on Tolgee. I guess the short version works here.